DataSet.rst

```diff
@@ -456,39 +480,48 @@ DataSet External Resource MetaTypes and File Extensions
 | ReferenceSet          | PacBio.ReferenceFile.ReferenceFastaFile        | .fasta                    |
 +-----------------------+------------------------------------------------+---------------------------+
 
-SubreadSet Special Purpose ExternalResources:
-    - PacBio.SubreadFile.ScrapsBamFile
-    - PacBio.SubreadFile.HqRegionBamFile
-    - PacBio.SubreadFile.HqScrapsBamFile
-    - PacBio.SubreadFile.LqRegionBamFile
-    - PacBio.SubreadFile.LqScrapsBamFile
-    - PacBio.SubreadFile.PolymeraseBamFile
-    - PacBio.SubreadFile.PolymeraseScrapsBamFile
-    - PacBio.SubreadFile.ChipStatsFile (.sts.xml)
-    - PacBio.SubreadFile.ChipStatsH5File (.sts.h5)
-    - PacBio.SubreadFile.AdapterFastaFile
-    - PacBio.SubreadFile.ControlFastaFile
-
-Bam-Related Special Purpose ExternalResources:
-    - PacBio.Index.BamIndex (.bam.bai)
-    - PacBio.Index.PacBioIndex (.bam.pbi)
-
-Fasta-Related Special Purpose ExternalResources:
-    - PacBio.Index.SamIndex (.fasta.fai)
-    - PacBio.Index.SaWriterIndex (.fasta.sa)
-    - PacBio.Index.Indexer (.fasta.index)
-    - PacBio.Index.FastaContigIndex (.fasta.contig.index)
-    - PacBio.GmapDB.GmapDBSummary (a file indicating the location of a gmap db)
+Some ``ExternalResource``'s themselves contain associated ``ExternalResources``'s, for example the
+indices associated with FASTA files. These associated files are nested within the primary ``ExternalResource``
+element to denote their subsidiary nature.
```

Comment:
Really the spirit here is to prevent an recursion of SubreadSets that contain SubreadSets. I'll change it to be of the same type, and see how far that gets us.